capabilities.json

```diff
@@ -361,6 +361,7 @@
                     "displayName": "Transparency",
                     "displayNameKey": "Visual_Transparancy",
                     "description": "Set transparency for background color",
+                    "descriptionKey": "Visual_Description_Transparancy",
```

Comment:
Please fix spelling mistake for all of properties. Transpar`e`ncy instead of Transpar`a`ncy